content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,181 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  Bring reproducibility and GPUs to your ML projects in Github or Gitlab pipelines with CML self-hosted runners.
```

Comment:
Throughout, let's use the format "GitHub" and "GitLab" instead of "Github" and "Gitlab".

Also for short description here's a suggestion:
``` Use your own GPUs with GitHub Actions & GitLab CI for automated machine learning.```